server/src/main/java/org/apache/druid/server/coordinator/DruidCoordinator.java

```diff
@@ -653,8 +683,8 @@ private void stopBeingLeader()
     private final long startTimeNanos = System.nanoTime();
     private final List<CoordinatorDuty> duties;
     private final int startingLeaderCounter;
-    private int cachedBalancerThreadNumber;
-    private ListeningExecutorService balancerExec;
+    //private int cachedBalancerThreadNumber;
```

Comment:
unused variables should be removed instead of commented

Reply:
oh, good catch, will remove them, thanks for reviewing